README.md

```diff
@@ -117,6 +117,60 @@ Here is a list of some of the key features being worked on for the upcoming rele
 
 Review detailed architecture in our [Docs](https://docs.yugabyte.com/latest/architecture/).
 
+# Codemap
+
+The YugabyteDB code is split into two top-level section:
+* [`postgres`](src/postgres/)
+<br /> Our modified fork of the Postgresql code. This is mostly C code.
```

Comment:
Instead of a break, just treating the links as run-in headings feels like it would work just as well, and be easier to read. Like this:
```suggestion
* [`postgres`](src/postgres/) is our modified fork of the PostgreSQL code. This is mostly C code.
```